MetaMask/metamask-extension
# UX: Updated Popular Networks List

This PR is to update the networks list by adding zkSync and Base as popular custom networks and Removing Aurora, Fantom, Harmony, and Palm networks.

## **Related issues**

Fixes: [#1601](https://github.com/MetaMask/MetaMask-planning/issues/1601)

## **Manual testing steps**

1. Go to Add Networks
2. Check Aurora, Fantom, Harmony, and Palm networks are no longer in the list
3. zkSync Era and Base should be there.
4. The Updated list of networks should look like

- Arbitrum One
- Avalanche C-Chain
- Base Mainnet
- BNB Chain Mainnet
- Celo Mainnet
- Ethereum Mainnet
- Linea
- Optimism
- Polygon Mainnet
- zkSync Era Mainnet


## **Screenshots/Recordings**


### **Before**
![Screenshot 2023-11-21 at 12 17 39 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/ea7b88e7-2960-4eb1-9777-61b0d30811df)


### **After**

![Screenshot 2023-11-21 at 12 18 19 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/bf330460-a340-4bca-8ce6-134537a0d8bd)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
